src/api.raml

```diff
@@ -795,6 +795,15 @@ traits:
                                 complete:
                                     displayName: Complete
                                     value: !include samples/collection/v2/complete.json
+                        application/vnd.elife.collection+json;version=3:
```

Comment:
this should be experimental and we could drop support for v1 at this stage and remove the experimental form v2.